### PR TITLE
Compose MapView:  ViewLabelProperties

### PR DESCRIPTION
**Pull request by shubham7109**
Related issue: `#2967`

### Description

PR to add the `labeling` property to the composable MapView.

### CODE REVIEW

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt [hunk 1]
```diff
@@ -60,6 +66,7 @@ public fun MapView(
     arcGISMap: ArcGISMap? = null,
     locationDisplay: LocationDisplay = rememberLocationDisplay(),
     mapViewInteractionOptions: MapViewInteractionOptions = MapViewInteractionOptionDefaults,
+    labeling: ViewLabelProperties = MapViewLabelingDefaults,
```
Comment (puneet-pdx) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:69]: add doc for labeling parameter.

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt [hunk 2]
```diff
@@ -36,13 +36,19 @@ import com.arcgismaps.mapping.ArcGISMap
...
 
 /**
  * The default instance of [MapViewInteractionOptions]
  */
 public val MapViewInteractionOptionDefaults: MapViewInteractionOptions = MapViewInteractionOptions()
 
+/**
+ * The default instance of [ViewLabelProperties]
+ */
+public val MapViewLabelingDefaults: ViewLabelProperties = ViewLabelProperties()
```
Comment (puneet-pdx) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:50]: `MapViewLabelPropertiesDefaults` ?

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt (hunk 1 above)
Comment (puneet-pdx) [reply toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:69]: Also should be called `viewLabelProperties` to keep it consistent here ? I know we have it as labeling in the sdk which might have been influenced by swift but does not make sense as a parameter here. 
@gunt0001 @shubham7109 thoughts ?

Comment (shubham7109) [reply toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:69]: I would think `viewLabelProperties` would be more clear. I'll use this suggestion in the PR.

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt [hunk 3]
```diff
@@ -44,6 +45,11 @@ import kotlinx.coroutines.launch
  */
 public val MapViewInteractionOptionDefaults: MapViewInteractionOptions = MapViewInteractionOptions()
 
+/**
+ * The default instance of [ViewLabelProperties]
+ */
+public val MapViewLabelPropertiesDefaults: ViewLabelProperties = ViewLabelProperties()
```
Comment (puneet-pdx) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:51]: same comment as [here](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/pull/173#discussion_r1376753661)

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt [hunk 4]
```diff
@@ -52,6 +58,7 @@ public val MapViewInteractionOptionDefaults: MapViewInteractionOptions = MapView
  * @param locationDisplay the [LocationDisplay] used by the composable [com.arcgismaps.toolkit.geocompose.MapView]
  * @param geometryEditor the [GeometryEditor] used by the composable [com.arcgismaps.toolkit.geocompose.MapView] to create and edit geometries by user interaction.
  * @param mapViewInteractionOptions the [MapViewInteractionOptions] used by this composable [com.arcgismaps.toolkit.geocompose.MapView]
+ * @param viewLabelProperties the [ViewLabelProperties] used by the composable [com.arcgismaps.toolkit.geocompose.MapView] to set the label property.
```
Comment (puneet-pdx) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:61]: `the [ViewLabelProperties] to be used by the composable [com.arcgismaps.toolkit.geocompose.MapView]` to display labels

**Review by puneet-pdx — APPROVED** [review]

**Review by gunt0001 — APPROVED** [review]
@shubham7109 - looks good.

**Pull request merged**